lib/endpoints/class-wp-rest-users-controller.php

```diff
@@ -517,6 +517,17 @@ public function prepare_item_for_response( $user, $request ) {
 		}
 
 		$context = ! empty( $request['context'] ) ? $request['context'] : 'embed';
+
+		if ( 'edit' === $context ) {
+			if ( empty( $data['capabilities'] ) ) {
+				$data['capabilities'] = new stdClass();
```

Comment:
I would lean towards empty object as the schema itself is object, or we can change the schema do be array( 'object', 'null' );  I will check to see if there is any info regarding what is better.